### DISCUSSION TITLE
What's the difference between karpenter and karpenter-core repositories?

### DISCUSSION THREAD

**Discussion by hawkesn** [kubernetes-sigs/karpenter#222] [Q&A] [answered]
What's the difference between karpenter and karpenter-core repositories?

**Comment by ellistarn** [comment, marked as answer]
https://github.com/aws/karpenter-core contains vendor neutral libraries and configurations, for use by other cloud providers. https://github.com/aws/karpenter contains the AWS specific cloud provider, binary, and helm chart.
